processing/src/main/java/org/apache/druid/segment/virtual/SingleStringInputDeferredEvaluationExpressionDimensionVectorSelector.java

```diff
@@ -118,7 +119,7 @@ public int getCurrentVectorSize()
    */
   private static final class StringLookupVectorInputBindings implements Expr.VectorInputBinding
   {
-    private final String[] currentValue = new String[1];
+    private final Object[] currentValue = new Object[1];
```

Comment:
Nitish: The naming of this private static class doesn't really line up with its implementation anymore?

Reply:
I guess it is still used exclusively for 'deferred' single string expressions, so maybe is ok?